src/status_im/contexts/chat/home/add_new_contact/views.cljs

```diff
@@ -109,8 +111,8 @@
 
 (defn new-contact
   []
-  (let [{:keys [public-key ens state msg]} (rf/sub [:contacts/new-identity])
-        customization-color                (rf/sub [:profile/customization-color])]
+  (let [{:keys [public-key ens state msg] :as i} (rf/sub [:contacts/new-identity])
```

Comment:
why as `:as i` ??
I don't think you're even using `i` and it's a slightly confusing name tbh given that `i` is usually for an index (afaik)